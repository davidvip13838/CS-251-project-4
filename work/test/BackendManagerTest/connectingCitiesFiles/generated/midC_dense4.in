500
1 32
11 16
20 0
13 19
18 44
15 25
39 41
37 1
17 35
7 32
37 31
10 6
24 41
1 1
11 12
9 31
42 34
50 14
43 5
50 7
41 27
21 36
4 3
36 44
4 16
36 33
28 20
42 40
50 0
31 9
27 5
28 18
31 5
50 36
28 30
2 42
36 28
29 32
22 9
47 20
49 4
24 5
49 21
45 33
43 30
25 9
12 18
0 14
43 15
31 32
13 40
30 14
8 10
45 12
43 42
8 29
4 0
16 3
37 30
45 15
4 25
15 41
35 2
50 48
19 45
38 13
45 0
22 30
33 43
8 43
18 29
4 2
0 36
47 2
44 34
16 33
17 26
49 45
4 24
28 2
32 31
50 8
35 14
30 7
19 48
17 3
50 16
37 43
10 22
33 10
45 13
2 26
16 45
47 50
21 25
21 30
34 12
45 8
16 25
49 32
25 44
48 35
47 5
20 35
20 9
39 23
23 32
9 7
49 40
34 23
8 14
24 34
41 45
6 20
8 32
34 32
38 3
44 22
17 29
26 3
15 50
27 17
41 35
43 10
23 31
12 20
25 49
50 46
29 21
36 48
41 14
11 35
7 27
3 15
3 22
8 35
19 22
27 28
41 16
47 29
38 50
20 46
19 50
32 35
6 45
37 50
28 0
16 17
41 12
39 10
1 4
12 27
2 39
14 37
36 27
26 17
29 4
46 21
10 39
14 41
20 37
16 43
19 35
49 3
34 48
20 39
12 31
0 13
47 34
12 1
8 17
47 17
18 6
36 17
18 32
40 42
35 23
1 7
41 17
44 33
6 13
10 18
3 50
25 31
8 19
33 11
15 20
23 29
13 3
45 29
17 42
11 43
5 11
16 22
15 43
38 23
6 21
43 29
32 33
12 4
15 44
16 30
7 39
15 40
12 35
10 4
18 31
1 24
2 47
7 18
25 39
32 16
7 24
35 48
23 26
50 2
31 16
21 43
34 42
7 15
18 3
15 1
29 16
16 26
39 8
41 3
48 28
4 9
20 3
36 39
49 13
24 15
13 18
19 46
35 5
49 17
41 50
25 33
14 43
24 27
46 50
40 14
13 24
11 10
47 49
29 37
42 9
42 6
27 21
32 47
36 50
34 44
20 42
15 37
17 5
20 5
44 45
26 14
7 9
11 25
46 27
38 26
0 28
44 16
38 9
21 7
22 5
17 14
47 38
2 38
20 45
14 22
36 14
2 41
10 47
22 48
28 11
25 37
16 39
23 17
28 3
28 40
11 7
8 23
41 15
30 47
48 4
24 29
37 42
19 39
41 28
37 37
23 33
13 2
50 35
5 2
5 0
48 18
10 32
19 21
39 38
14 24
33 38
32 21
45 7
7 5
50 39
39 24
2 8
6 35
43 20
48 45
34 49
26 8
48 33
21 49
49 50
8 2
29 31
39 7
4 23
32 8
33 1
20 8
15 36
0 5
6 37
6 38
16 24
6 50
23 8
22 21
38 40
39 22
37 47
39 6
35 41
13 33
34 40
27 50
15 21
10 9
15 24
18 19
16 37
43 39
20 6
29 42
40 32
38 18
8 48
42 15
2 20
49 5
8 49
44 48
46 9
12 8
31 17
45 44
42 41
37 45
5 40
6 26
50 15
45 30
1 40
32 36
38 45
10 7
20 50
18 37
7 0
37 15
49 20
16 12
19 17
37 48
11 24
46 30
6 1
34 22
49 29
24 13
17 46
42 12
22 20
48 11
14 16
32 44
26 48
48 50
16 47
39 25
25 4
9 48
34 31
20 23
19 0
42 14
4 5
49 15
46 43
42 2
19 9
12 32
0 17
4 48
1 36
13 0
42 11
40 19
46 37
35 16
33 33
5 22
17 18
34 34
3 27
20 13
13 35
2 4
36 6
24 36
31 46
20 27
47 16
13 43
2 0
26 7
41 47
27 10
9 11
18 41
22 36
26 28
14 49
11 22
15 47
35 42
39 26
40 16
33 15
16 32
48 9
20 4
2 31
8 18
47 39
35 13
27 11
28 8
0 46
46 2
39 3
5 10
3 14
2 3
9 25
7 10
13 15
4 41
21 3
14 21
13 31
47 3
11 21
32 4
28 45
0 50
49 47
12 48
44 37
12 0
16 21
22 27
50 23
9 17
48 24
22 4
16 8
27 36
7 43
24 22
29 46
27 48
23 28
13 14
39 40
15 19
14 19
31 42
35 25
25 18
34 47
47 32
24 49
26 49
7 36
32 37
