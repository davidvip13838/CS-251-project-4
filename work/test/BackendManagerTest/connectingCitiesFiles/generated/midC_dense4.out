790.7771441734415